src/pretix/control/forms/mailsetup.py

```diff
@@ -54,6 +56,15 @@ class SMTPMailForm(SettingsForm):
     smtp_password = SecretKeySettingsField(
         label=_("Password"),
         required=False,
+        validators=[RegexValidator(
+            r"^[A-Za-z0-9!\"#$%&'()*+,-./:;<=>?@\^_`{}|~]+$",
```

Comment:
```suggestion
            r"^[A-Za-z0-9!\"#$%&'()*+,./:;<=>?@\^_`{}|~-]+$",
```
`-` should always be last in a character class, otherwise it specifies a range. However this case, the result is accidentally (or trickily) correct, because
```
>>> ord(",")
44
>>> ord("-")
45
>>> ord(".")
46
```

Reply:
I'd love to say this was intentional, which it kinda was since I typed the characters down from an ASCII table, but let's better not leave it as a work of art :D